NAMESPACE

```diff
@@ -2,88 +2,47 @@
 
 export(FtestCNV)
 export(adjust_ploidy)
-export(annotate_hotspots)
 export(annotate_maf_triplet)
-export(assign_cn_to_ssm)
-export(calc_mutation_frequency_bin_region)
-export(calc_mutation_frequency_bin_regions)
-export(calc_mutation_frequency_sliding_windows)
 export(calculate_pga)
 export(cleanup_maf)
 export(cnvKompare)
-export(collate_lymphgen)
-export(collate_results)
-export(consolidate_lymphgen)
 export(count_ssm_by_region)
-export(estimate_purity)
+export(fread_maf)
 export(gene_to_region)
 export(genome_to_exome)
-export(get_bams)
-export(get_cnv_and_ssm_status)
-export(get_coding_ssm_status)
-export(get_sample_wildcards)
-export(get_samples_md5_hash)
-export(get_ssh_session)
-export(id_ease)
 export(intersect_maf)
 export(maf_to_custom_track)
 export(make_igv_snapshot)
 export(region_to_bins)
 export(region_to_gene)
-export(review_hotspots)
 export(sanitize_maf_data)
 export(supplement_maf)
 export(sv_to_bedpe_file)
 export(sv_to_custom_track)
 export(tidy_lymphgen)
 export(view_mutation_igv)
-export(write_sample_set_hash)
 import(ComplexHeatmap)
-import(DBI)
-import(RMariaDB)
+import(GAMBLR.helpers)
 import(Rsamtools)
```

Comment:
Should `GAMBLR.viz` also be imported here? Or is it sufficient to only import the two functions with `importFrom` (a bit further down this script.)

Reply:
I have added it here, thanks!